locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -1271,6 +1267,7 @@ msgctxt "Some hero killed by some heroes"
 msgid "killed by"
 msgstr "死于"
 
+# I can't find this on site.
```

Comment:
You can find this string on the match page with replay, on the Statistics tab, Match log tab, when you mouse over on a killed hero.